src/components/landing/LandingPage.css

```
@@ -51,6 +109,145 @@
+}
+
+.hero-window-card {
+  animation: hero-window-drift 10s ease-in-out infinite;
+  transform-origin: center;
+  will-change: transform, translate, scale;
+}
+
+.hero-float-a {
+  --float-x: 3px;
+  --float-y: -4px;
+  animation-delay: -1.6s;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)